README.md

```diff
@@ -45,7 +45,20 @@ For more info on postgres commands, see https://www.postgresql.org/docs/
 
 ## Usage Instructions
 
-TODO: write these
+### Seed the catalog
+
+Seeding the catalog presumes an empty or nearly empty database -- otherwise running the seed task will throw `druid NOT expected to exist in catalog but was found` errors for each found object. You can monitor the progress of the seed task by tailing `log/production.log`. The seed task can take a while -- check [the repo wiki for stats](https://github.com/sul-dlss/preservation_catalog/wiki/Stats) on the timing of past runs, and you can expect profiling runs will add overhead. The following rake tasks should be run from the root directory of the project, with whatever `RAILS_ENVIRONMENT` is appropriate.
+
+Without profiling:
+```ruby
+RAILS_ENVIRONMENT=production bundle exec rake seed_catalog
+```
+
+With profiling:
+```ruby
+RAILS_ENVIRONMENT=production bundle exec rake seed_catalog[profile]
+```
```

Comment:
since this could be a job that potentially runs for days, if running it over all prod storage roots, i'd maybe make two other practical suggestions for users:
* run it from a screen session (random cheatsheet/tutoral that i've found helpful for screen:  http://thingsilearned.com/2009/05/26/gnu-screen-super-basic-tutorial/)
* run it in the background under nohup:  `RAILS_ENV=production nohup bundle exec rake seed_catalog &`

Reply:
I agree w John.